### PR TITLE
Add Wilbrand Web instructions to the Wilbrand page

### PR DESCRIPTION
**Pull request by jornmann**
Wilbrand Web is a web frontend for Wilbrand that I recently wrote, you can see it at [https://wilbrand.donut.eu.org/](https://wilbrand.donut.eu.org/). This way, it is as easy to use as LetterBomb.
I thought it would be a good idea to add instructions on how to use it to wii.guide.
This is my first contribution to wii.guide, so let me know if I made any mistakes.
Thank you!

### CODE REVIEW

File: _pages/en_US/wilbrand.md [hunk 1]
```diff
@@ -10,8 +10,44 @@ If you need help for anything regarding this tutorial, please join [the RiiConne
...
+- An SD card formatted to FAT32/MS-DOS
+- A Wii on at least version 3.0
+
+SD cards larger than 2GB will not work on Wii menu versions before 4.0.
+{: .notice--warning}
+
+#### Instructions
+
+1.  On your Wii, go to `Wii Settings`, and make note of the version at the top right. Proceed to `Internet` -> `Console Information` and also make note of your MAC address.
+1.  Visit [wilbrand.donut.eu.org](https://wilbrand.donut.eu.org/), input your Wii MAC, version, region, ensure `Bundle the HackMii Installer for me!` is checked, and cut either wire.
```
Comment (Naim2000) [inline _pages/en_US/wilbrand.md:31]: By the way you have the readme's bundled with the HackMii installer? I'm moving in a minute so I can't check

Comment (jornmann) [reply _pages/en_US/wilbrand.md:31]: yes, the readmes and licenses are added along with the installer

File: _pages/en_US/wilbrand.md [hunk 2]
```diff
@@ -10,8 +10,44 @@ If you need help for anything regarding this tutorial, please join [the RiiConne
 Wilbrand, like LetterBomb, is an exploit for the Wii that is triggered using the Wii Message Board.
 
 Unlike LetterBomb, Wilbrand supports Wii menu versions down to 3.0.
+
```
Comment (Naim2000) [inline _pages/en_US/wilbrand.md:13]:
```suggestion
Unlike LetterBomb, Wilbrand supports Wii menu versions down to 3.0.
{: .notice--info}

```
Where'd this go

File: _pages/en_US/wilbrand.md [hunk 3]
```diff
@@ -10,8 +10,44 @@ If you need help for anything regarding this tutorial, please join [the RiiConne
...
+1.  Take out your SD card and insert it in your Wii.
+    - The SD card must be inserted in the SD card slot located in the front of the Wii. Using a USB adapter plugged into the Wii's USB port will not work.
+1.  On your Wii, return to the Wii Menu and then open the Wii Message Board.
+1.  Load the green letter with the Bob-omb icon.
+    - Ensure the date on your Wii is correct, otherwise you might be unable to find the letter.
+    - The mail may have appeared on the previous or next day.
+    - If you don't see the green letter, check if any errors appear in the SD card sections in `Data Management`. If any do, there may be an issue with the SD card format or the Wii's SD card reader.
+      ![Wilbrand in its natural habitat](/images/wilbrand/msgboard.png)
+
+If this didn't work for you, try another exploit or try Wilbrand.
```
Comment (Naim2000) [inline _pages/en_US/wilbrand.md:44]:
```suggestion
If this didn't work for you, [try another exploit](get-started) or try the normal [Wilbrand](#wilbrand).
```
Hyperlinks are built like that
oh and "or try Wilbrand" sounds kinda silly, like this is Wilbrand in the first place

**Review by Naim2000 — CHANGES_REQUESTED** [review]

File: _pages/en_US/wilbrand.md (hunk 3 above)
Comment (playerjmr) [reply _pages/en_US/wilbrand.md:44]:
```suggestion
If this didn't work for you, [try another exploit](get-started) or try [Wilbrand CLI](#wilbrand).
```

Comment (Naim2000) [reply _pages/en_US/wilbrand.md:44]: Fair

**Review by Naim2000 — APPROVED** [review]
Pro tip: use separate branches for your pull requests, I squash a lot

**Pull request merged**